src/py42/exceptions.py

```diff
@@ -25,29 +24,29 @@ class Py42ArchiveFileNotFoundError(Py42ResponseError):
     """An exception raised when a resource file is not found or the path is invalid."""
 
     def __init__(self, response, device_guid, file_path):
-        message = u"File not found in archive for device {} at path {}".format(
+        message = "File not found in archive for device {} at path {}".format(
```

Comment:
This would probably be better as an fstr

Reply:
Same with all the exception message construction in this module, seems like the autoformatted did not do it but I think it would be better to use `f-str`s here in this file